hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/HoodieMetrics.java

```diff
@@ -274,6 +274,17 @@ String getMetricsName(String action, String metric) {
     return config == null ? null : String.format("%s.%s.%s", config.getMetricReporterMetricsNamePrefix(), action, metric);
   }
 
+  public void updateClusteringFileCreationMetrics(long durationInMs) {
+    metrics.registerGauge(getMetricsName("replacecommit", "fileCreationTime"), durationInMs);
+  }
+
+  /**
+   * Given a commit action, metrics name and value this method publishes custom metrics.
+   */
+  public void publishMetrics(String commitAction, String metricsName, long value) {
```

Comment:
```suggestion
  public void reportMetrics(String commitAction, String metricsName, long value) {
```

Reply:
Thanks for catching this, changed the param and method name.